From b9d260a49961b5e311e77e8600e7e5a8f2e8d475 Mon Sep 17 00:00:00 2001
From: Keith Packard <keithp@keithp.com>
Date: Tue, 1 May 2012 21:53:02 -0700
Subject: [PATCH] keystone.5c: cairo-5 box semantics changed default layout

cairo-5c version 1.6 includes a new 2D box layout widget that
needs widget stretch defined in both dimensions to create
the desired layout.

Signed-off-by: Keith Packard <keithp@keithp.com>
---
 keystone.5c |    9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/xrandr/keystone.5c b/xrandr/keystone.5c
index c616f12..4b698ee 100644
--- a/xrandr/keystone.5c
+++ b/xrandr/keystone.5c
@@ -543,15 +543,14 @@ void main ()
     &box_t	hbox = Box::new (Box::dir_t.horizontal,
 				 Box::widget_item (&doit, 0),
 				 Box::widget_item (&show, 0),
-				 
 				 Box::widget_item (&quit, 0),
 				 Box::glue_item (1));
     &box_t	box = Box::new (Box::dir_t.vertical,
 				Box::box_item (&hbox),
-				Box::widget_item (label[0], 0),
-				Box::widget_item (label[1], 0),
-				Box::widget_item (label[2], 0),
-				Box::widget_item (&space, 0),
+				Box::widget_item (label[0], 1, 0),
+				Box::widget_item (label[1], 1, 0),
+				Box::widget_item (label[2], 1, 0),
+				Box::widget_item (&space, 1, 0),
 				Box::widget_item (&quad, 1));
     Nichrome::set_box (&nichrome, &box);
     Nichrome::main_loop (&nichrome);
-- 
1.7.10

